diff --git a/openbox/client.c b/openbox/client.c
index f110a66..9746808 100644
--- a/openbox/client.c
+++ b/openbox/client.c
@@ -2623,6 +2623,8 @@ gboolean client_is_oldfullscreen(const ObClient *self,
 {
     const Rect *monitor, *allmonitors;
 
+    return FALSE;
+
     /* No decorations and fills the monitor = oldskool fullscreen.
        But not for maximized windows.
     */

#,...,...,.,.,,.,,..,,,.,,...,,,,,,..,.,.,.,,,..,,,,.,,..,,.,,.,,,,,,,..,,...,
#H5FAOMNJ7P46JJRDUK5PSIX6I5UJBEY2ZB62ANAQYLQ77CAV3EKQ7UXFBPADK3KNYS74MEHRZ5VXY
#\\\|66JLRJ6NNI4XOBEZILYPHNZWZFX73K2CJQVGLLZSTNCAZFHFFH4 \ / AMOS7 \ YOURUM ::
#\[7]A4IZVQZ3PPCHTQKX3KJ5KSXN3JABQBERN2FPHVBNZUZMRZ4WUCAQ 7  DATA SIGNATURE ::
#:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
