diff --git a/openbox/client.c b/openbox/client.c
index f110a66..9746808 100644
--- a/openbox/client.c
+++ b/openbox/client.c
@@ -2623,6 +2623,8 @@ gboolean client_is_oldfullscreen(const ObClient *self,
 {
     const Rect *monitor, *allmonitors;
 
+    return FALSE;
+
     /* No decorations and fills the monitor = oldskool fullscreen.
        But not for maximized windows.
     */

#,,.,,,.,,,,.,,.,,.,,,,..,,..,.,.,.,.,,,,,,.,,.,.,...,...,,..,.,,,.,.,.,,,,,,,
#H4WD73AB4AXEDVPA2XUDZLGSHMWZHAU2PT4MOH3C2D4WAHQDHVGQQUJ5MM4F3SM5R6UOZK6BPCEMQ
#\\\|Y4XN5HTIQLFX2UEIWVZ2UYIC2JW44XIRVQ7KN6G2HYUHH6CUE2I \ / AMOS7 \ YOURUM ::
#\[7]NQHGD7FT5CCKUJDRTZCIMGVKCLDJ6R73XT6FPUYSCQCXGUVRDCAY 7  DATA SIGNATURE ::
#:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
