diff --git a/openbox/client.c b/openbox/client.c
index f110a66..9746808 100644
--- a/openbox/client.c
+++ b/openbox/client.c
@@ -2623,6 +2623,8 @@ gboolean client_is_oldfullscreen(const ObClient *self,
 {
     const Rect *monitor, *allmonitors;
 
+    return FALSE;
+
     /* No decorations and fills the monitor = oldskool fullscreen.
        But not for maximized windows.
     */

#,,.,,,.,,,,.,,.,,.,,,,..,,..,.,.,.,.,,,,,,.,,..,,...,...,..,,,,,,,,,,,.,,..,,
#4MHTZJ2BYBGG3WX2A7QQIT3YEKXNMB6MXWUNY3EISA55GINPH5BDWABDWARQ3SZTRODACWVCHWUJG
#\\\|BDC4ACOH5LG5MRQ3M7Z6PAAS5TZ35HDE2HLWD5A5SGIQEWRJDDF \ / AMOS7 \ YOURUM ::
#\[7]7YR4HVUKNZ6OMOMMI6SXH4JB74IX5PAFZN6LQEDGO5ZLVAEGHECQ 7  DATA SIGNATURE ::
#:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
