diff --git a/openbox/client.c b/openbox/client.c
index f110a66..9746808 100644
--- a/openbox/client.c
+++ b/openbox/client.c
@@ -2623,6 +2623,8 @@ gboolean client_is_oldfullscreen(const ObClient *self,
 {
     const Rect *monitor, *allmonitors;
 
+    return FALSE;
+
     /* No decorations and fills the monitor = oldskool fullscreen.
        But not for maximized windows.
     */

#.............................................................................
#D7L7HE7M6UZM6GH4IOT7WMKBMGWH635YUQPHAL52PTVG3XV26NFFQABMFLWJUYGS5BR4IKEWZDTPS
#::: VIVN36JEKDDUGYZ64GQNSFXAVAW2VSHGRURQOYV3KMGEO4QK3OF :::: NAILARA AMOS :::
# :: FRKMJLOMAWDPCU5O67D5PWEGFKTFQ2TZIJI3UPOSJ5S2B6EXGYAI :: CODE SIGNATURE ::
# ::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
