diff --git a/openbox/client.c b/openbox/client.c
index f110a66..9746808 100644
--- a/openbox/client.c
+++ b/openbox/client.c
@@ -2623,6 +2623,8 @@ gboolean client_is_oldfullscreen(const ObClient *self,
 {
     const Rect *monitor, *allmonitors;
 
+    return FALSE;
+
     /* No decorations and fills the monitor = oldskool fullscreen.
        But not for maximized windows.
     */

#,,.,,,.,,,,.,,.,,.,,,,..,,..,.,.,.,.,,,,,,.,,.,.,...,..,,.,,,.,,,,.,,...,...,
#54C2RPZGUKFVCFMIN7ERZBCBGFNS6DIJYJOSSF76THSBOYHYJEEMZEGBTSESPGR4JFXQSK5BAGJYW
#\\\|3Q6XDBB2MR2B72ZISZ2KSG3CMNFU3AEG3TC6GBTYKWYFAZCYM75 \ / AMOS7 \ YOURUM ::
#\[7]TR5UZENUJST7FYXHSXAVTKK5FYQLWA4J3EIUS6XSW6CHRHQMMIDA 7  DATA SIGNATURE ::
#:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
